engine/lib/concrete_ident/concrete_ident.ml

```diff
@@ -547,27 +553,48 @@ module MakeRenderAPI (NP : NAME_POLICY) : RENDER_API = struct
           ((`Type n | `Const n | `Fn n), (`Trait _ | `Impl (_, `Trait, _))) ->
           prefix "f" (dstr n)
```

Comment:
It seems the update of how names are rendered broke the Coq naming scheme, since the trait name needs to be prepended. Changing this to 
```ocaml
      | `AssociatedItem
          ((`Type n | `Const n | `Fn n), (`Impl (_, `Trait, _))) ->
        prefix "f" (dstr n)
      | `AssociatedItem
          ((`Type n | `Const n | `Fn n), (`Trait (a, _))) ->
          Concat (dstr a, prefix "f" (dstr n))
```
Only half fixes the issue, as I still need to prefix the impl function name.